lightning/src/ln/channelmanager_limits_tests.rs

```diff
@@ -0,0 +1,710 @@
+use bitcoin::hashes::Hash;
+use crate::ln::channelmanager::{PaymentId, PaymentSendFailure, RecipientOnionFields};
+use crate::ln::functional_test_utils::*;
+use crate::util::errors::APIError;
+use bitcoin::secp256k1::{PublicKey, Secp256k1, SecretKey};
+use crate::events::{Event, MessageSendEvent, MessageSendEventsProvider, ClosureReason};
+use crate::ln::channelmanager;
+use crate::ln::ChannelId;
+use crate::ln::msgs::{self};
+use crate::ln::msgs::ChannelMessageHandler;
+use crate::prelude::*;
+use crate::util::config::ChannelConfig;
+use crate::sign::EntropySource;
```

Comment:
nit: alphabetize these imports. I think if you copy this into the rust playground and run `rustfmt` it will do it for you. Please check the other new test files in this commit to see if they need it too.